db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/SparkRelationalOperator.java

```diff
@@ -66,9 +68,9 @@ public SparkRelationalOperator(int relOpKind,
     @Override
     public Column getColumnExpression(Dataset<Row> leftDF,
                                       Dataset<Row> rightDF,
-                                      ParserInterface parser) throws UnsupportedOperationException {
```

Comment:
Could you use a more descriptive name for the function? Maybe parser, parserFunction or something like that?

Reply:
OK.  Changing it to convertStringToDataTypeFunction.